scripts/authTokenGenerator.py

```diff
@@ -34,16 +34,16 @@ def fetchAuthToken(email, password):
 	refreshToken = resJson['refresh_token']
 
 	# persist auth info
-	with open('./assets/auth', 'w') as auth_file:
+	with open('/tmp/teslaAuthFile', 'w+') as auth_file:
```

Comment:
@aditprab is there a reason we need to keep expired tokens?

Reply:
disregard this. Just a gap in my python knowledge. I thought `w+` appended to the file.